hack/deploy-cnv.sh

```diff
@@ -98,5 +98,10 @@ spec:
   BareMetalPlatform: true
 EOF
 
+if [[ "${KUBEVIRT_RELEASE}" =~ 0.34 ]]; then
```

Comment:
Also, I wonder about the significance of creating the ConfigMap only after the `HyperConverged` CR is created, and consequently, The `KubeVirt` CR might have been already created too.

From virt-controller/virt-operator's perspective (or whoever consumes this ConfigMap), is this significant?